okpt/io/utils/writer.py

```diff
@@ -37,7 +38,7 @@ def get_file_obj(path: str) -> TextIOWrapper:
     return open(path, 'w')
 
 
-def write_json(data, file):
+def write_json(data: Dict[Any, Any], file: TextIOWrapper):
```

Comment:
Why did the python files get changed in this PR?

Reply:
The file was missing a few things, so I threw them in. But you're right, these changes aren't related to the PR and are better suited for another PR. I'll remove these changes.

Reply:
Makes sense, thanks